src/ansys/mapdl/core/_commands/graphics/labeling.py

```diff
@@ -0,0 +1,1247 @@
+        displayed for the arrows. The color of these arrows does not correspond to the magnitudes in the
+        contour legend. The length of these arrows does, however, correlate to the relative magnitude of the
+        pressures.
+
+        For elements SURF159, SOLID272, SOLID273, PIPE288 and PIPE289, :ref:`psf` is not available when
+        displaying elements with shapes determined from the real constants or section definition (
+        :ref:`eshape` ). For PIPE288 and PIPE289, only external loads applied via :ref:`sfbeam` are
+        displayed.
+
+        This command is valid in any processor.
+
+        .. _psf.tab.1:
+
+        **/PSF - Valid Item and Component Labels**
+
+        .. flat-table::
+           :header-rows: 1
+
+           * - Item
+             - Comp
+             - Description
+             - Comments
+           * - :rspan:`14` PRES [ :ref:`psf.labels.fn.1`]
+             - NORM (or blank)
+             - Applied pressure normal to the face (real component only).
+             - For element types other than  SURF153, SURF154 and SURF156.
+           * - NORM
+             - Applied pressure normal to the face (real component).
+             - :rspan:`5` For element types SURF153, SURF154 and SURF156 with KEYOPT(2) = 0. For `supported structural elements<https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cmd/Hlp_C_SFCONTROL.html#>`_ with``KCSYS`` = 0. [ :ref:`psf.labels.fn.2`]
+           * - TANX
+             - Applied pressure in the element tangential x direction (real component).
+           * - TANY
+             - Applied pressure in the element tangential- y direction (real component).
+           * - INRM
+             - Applied pressure normal to the face (imaginary component).
+           * - ITNX
+             - Applied pressure in the element tangential x direction (imaginary component).
+           * - ITNY
+             - Applied pressure in the element tangential y direction (imaginary component).
+           * - LOCX
+             - Applied pressure in the local x direction (real component).
+             - :rspan:`5` For element types SURF153, SURF154 and SURF156 with KEYOPT(2) = 1. For `supported structural element`_ with``KCSYS`` = 1. [ :ref:`psf.labels.fn.2`]
+           * - LOCY
+             - Applied pressure in the local y direction (real component).
+           * - LOCZ
+             - Applied pressure in the local z direction (real component).
+           * - ILCX
+             - Applied pressure in the local x direction (imaginary component).
+           * - ILCY
+             - Applied pressure in the local y direction (imaginary component).
+           * - ILCZ
+             - Applied pressure in the local z direction (imaginary component).
+           * - RVEC
+             - Applied pressure in the user-defined direction (real component).
+             - :rspan:`1` For `supported structural element`_ with``KCSYS`` = 2. [ :ref:`psf.labels.fn.2`]
```

Comment:
Link is missing here `supported structural element`.

Also the `[:ref:....]` is broken/looks weird:

<img width="566" alt="image" src="https://github.com/user-attachments/assets/547ceaa4-e9f5-4740-ada0-e7859eaac8f2" />

Reply:
I tried different approaches in order to fix the link rendering but I have found the solution yet. I opened an [issue](https://github.com/ansys/pyconverter-xml2py/issues/416) to keep a track of it.